modules/data-streams/src/test/java/org/elasticsearch/datastreams/lifecycle/health/DataStreamLifecycleHealthIndicatorServiceTests.java

```diff
@@ -167,6 +168,6 @@ public void testMultiProject() {
     }
 
     private HealthInfo constructHealthInfo(DataStreamLifecycleHealthInfo dslHealthInfo) {
-        return new HealthInfo(Map.of(), dslHealthInfo, Map.of());
+        return new HealthInfo(Map.of(), dslHealthInfo, Map.of(), FileSettingsHealthInfo.INDETERMINATE);
```

Comment:
I see you have to add this constructor parameter in a few classes that are unrelated to this change. I think we should come up with a different approach that avoids us having to keep making these changes every time we add a new item to the health cache. But that's not within the scope of this PR so I'm fine leaving it like this for now - I'm mostly making a note.

Reply:
💯